komodo/yaml_file_types.py

```diff
@@ -197,21 +197,36 @@ def validate_package_entry(
         self, package_name: str, package_version: str
     ) -> komodo_error:
         repository_entries = self.content
-        if package_name not in repository_entries:
+        if package_name not in repository_entries.keys():
+            for real_packagename in [
+                lowercase_pkg
+                for lowercase_pkg in repository_entries.keys()
+                if lowercase_pkg.lower() == package_name.lower()
+            ]:
+                raise KomodoException(
+                    f"Package '{package_name}' not found in repository. Did you mean"
+                    f" '{real_packagename}'?"
+                )
             raise KomodoException(f"Package '{package_name}' not found in repository")
         if package_version not in repository_entries[package_name]:
+            if f"v{package_version}" in repository_entries[package_name]:
+                raise KomodoException(
+                    f"Version '{package_version}' of package '{package_name}' not found"
+                    f" in repository. Did you mean 'v{package_version}'?"
```

Comment:
That `v` in front of `package_version` should perhaps not be there? I think there are version tags in komodo both including and not including that "v", so for some packages you might get two v's with this.

Reply:
If you want version v1.2.3 but type 1.2.3 instead, this will notify you that the existing version does not exist, but that you might mean v1.2.3 which does exist. We do the same for when the entered package_name's capitalisation does not match the one in repositoryyaml. Example PYTest instead of pytest.

Reply:
ah, I think I missed the `v` in line 212.